spec.html

```diff
@@ -44212,15 +44212,15 @@ <h1>
           1. If _isLittleEndian_ is *false*, reverse the order of the elements of _rawBytes_.
           1. If _type_ is ~float16~, then
             1. Let _value_ be the byte elements of _rawBytes_ concatenated and interpreted as a little-endian bit string encoding of an IEEE 754-2019 binary16 value.
-            1. If _value_ is an IEEE 754-2019 binary16 NaN value, return the *NaN* Number value.
+            1. If _value_ is an IEEE 754-2019 binary16 format Not-a-Number encoding, return *NaN*.
```

Comment:
Current IEEE 754 doesn't capitalize or hyphenate this phrase, though I see we do refer to it with this spelling elsewhere. Also, IEEE consistently uses "NaN", with "not a number" defined in the glossary as "see NaN".

As long as we're going for consistency, I'd prefer to change our existing uses of "Not-a-Number" to "NaN".

Reply:
That works for me. @syg do you have a preference?

Reply:
Alright, I changed them. There's still 2 uses of "Not-a-Number" in the Terms and Definitions section, which I'm not too worried about because it's helpful there.